### PR TITLE
Fix SSR issues

### PR DESCRIPTION
**Pull request by soroushchehresa**

### CODE REVIEW

File: src/load-js.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 function createLoadJS() {
+  if (typeof document === "undefined" || typeof window === "undefined") {
```
Comment (MiguelCastillo) [inline src/load-js.js:2]: Oh thank you so much!!

**Comment by MiguelCastillo** [comment]
@soroushchehresa Thank you so much for these changes!!  The built artifacts don't need to be checked in.  These are generated when I publish to npm, which I can do shortly once those two files are removed.  But Im happy to hear why you think these file should be checked into source control.

**Comment by soroushchehresa** [comment]
@MiguelCastillo Don't mention it 🙏.
You're right, because of I installed this library from [forked repo](https://github.com/soroushchehresa/load-js) in my `package.json` I needed build files and when I created pull request I forgot to add the build files to `.gitignore` again.
Thanks for your attention.

**Comment by MiguelCastillo** [comment]
oh gotcha! thank you :)

**Pull request closed without merging**